samples/client/petstore/php/SwaggerClient-php/pom.xml

```diff
@@ -46,7 +46,7 @@
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <executable>vendor/bin/phpunit</executable>
+                            <executable>vendor/bin/phpunit tests</executable>
```

Comment:
You need to do something like the following instead:
```
                        <configuration>
                            <executable>vendor/bin/phpunit</executable>
                            <arguments>
                                <argument>tests</argument>
                            </arguments>
                        </configuration>
```

Reply:
Thanks 😭 ✨